pkg/virt-operator/application.go

```diff
@@ -382,11 +397,17 @@ func (app *VirtOperatorApp) Run() {
 				OnStartedLeading: func(ctx context.Context) {
 					leaderGauge.Set(1)
 					log.Log.Infof("Started leading")
+
+					log.Log.V(5).Info("start monitoring the kubevirt-config configMap")
+					obsoleteCmTicker.Reset(obsoleteCmFrequency)
+
 					// run app
 					go app.kubeVirtController.Run(controllerThreads, stop)
 				},
 				OnStoppedLeading: func() {
 					leaderGauge.Set(0)
+					log.Log.V(5).Info("stop monitoring the kubevirt-config configMap")
```

Comment:
You don't have to stop anything. If we loose the election and we just exit asap. Even this `leaderGauge.Set(0)` very likely has 0 effect and could be removed.

Reply:
But with the current design, the goroutine keeps running. 

The problem is that the ticker's channel is read only and can't be closed.

Reply:
you don't need a ticker. Just make stop channel (you don't even ever need to stop this as I wrote).

Reply:
Can't I relay on the existing stop channel?

Reply:
Sure :+1: